events/beam-robotics.html

```diff
@@ -167,6 +167,10 @@ <h2 class="title text-1.5x">
       After learning the basics of programming our robot last week, this time we'll be coding our robot to dance! 
     </p>
   </div>
+  <div class="button-container">
+    <a class="button button-dark" href="https://docs.google.com/presentation/d/19gtmK_tlAnNI2G9sZNMvCPZzDapiBUVpstaGXEk4QoM/edit#slide=id.p" target="_blank" rel="noopener noreferrer"> <span class="fas fa-book"></span> Student Guide</a>
```

Comment:
ahh its not a student guide, its the slides for the day!

Reply:
Oh ok let me fix that real quick! Out of these two icons, which do you think is better for the slides?
https://fontawesome.com/icons/slideshare?style=brands
https://fontawesome.com/icons/file-powerpoint?style=solid

Reply:
i think we generally do this? https://6098d4d7c013c600080b22a9--acm-teach-la-website.netlify.app/citylab-cyber

Reply:
Oh ok, thanks!